pkg/dispatchcli/cmd/install.go

```diff
@@ -21,10 +21,10 @@ import (
 
 	"github.com/ghodss/yaml"
 	"github.com/imdario/mergo"
-	homedir "github.com/mitchellh/go-homedir"
+	"github.com/mitchellh/go-homedir"
```

Comment:
Is this a lint issue?  Because I kind of like the explicit package name where the name differs from the package URI.

Reply:
It was highlighted for me (might not be from golint but some other linter).